scraper/src/mindtouch2zim/constants.py

```diff
@@ -16,6 +16,8 @@
 HTTP_TIMEOUT_NORMAL_SECONDS = 15
 HTTP_TIMEOUT_LONG_SECONDS = 30
 
+WARN_ONLY_ON_HTML_ISSUES = False
```

Comment:
`WARN_ONLY_ON_HTML_ISSUES` means to only add warning when there are HTML issues

```suggestion
HTML_ISSUES_WARN_ONLY = False
```

This means what's intended (or closer to meaning than the previous version)